kommunicateui/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
     api 'com.theartofdev.edmodo:android-image-cropper:2.7.0'
     api 'com.github.bumptech.glide:glide:4.9.0'
     api 'com.android.support:cardview-v7:27.1.1'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
```

Comment:
@shubhamtewari can you check what will be new size addition on SDK after adding this?

Reply:
the entire sdk on adding 6 resources files and the new layout, increases by 140kb. The resource files are around 80kb